core/embed/kernel/main.c

```diff
@@ -92,7 +95,7 @@ void drivers_init() {
   dma2d_init();
 #endif
 
-  display_init(DISPLAY_RETAIN_CONTENT);
+  display_init(DISPLAY_JUMP_BEHAVIOR);
```

Comment:
@TychoVrahe What about renaming this to something that includes the word "handover" (to be consistent with the symbol in the linker script)? Something like `DISPLAY_CONTENT_AFTER_HANDOVER`... It’s a bit weird, but it might better convey the intent.

Reply:
hmm the functions we use to move between stages are still called jump, so there gonna be inconsistencies anyway until we can rename it all. I slightly prefer to keep current name until we do that

Reply:
Ok, I have no problem with this, it's a really minor issue. I think we will be revisiting `utils`/`limited_utils.s` soon and than we can reconsider it again.